lib/lambda/sinkMain.test.ts

```diff
@@ -0,0 +1,87 @@
+import { describe, it, expect, vi, afterEach } from "vitest";
```

Comment:
I'm wondering if we could use a function for the handler setup. Something like-
```
function createHandlerEvent(record) {
  return {
    eventSource: "SelfManagedKafka",
    bootstrapServers: "kafka",
    records: { [`"${record}"`]: [] },
  };
}
```

and then in the tests-
```
await handler(createHandlerEvent("aws.onemac.migration.cdc-0"), expect.anything(), vi.fn());
```

Reply:
Great suggestion: addressed with https://github.com/Enterprise-CMCS/macpro-mako/pull/897/commits/14a1721325400e22a2d343b6d9a2e55563708051